dbms/src/Storages/StorageDisaggregatedRemote.cpp

```diff
@@ -95,7 +96,11 @@ BlockInputStreams StorageDisaggregated::readFromWriteNode(
         try
         {
             auto remote_table_ranges = buildRemoteTableRanges();
-            auto batch_cop_tasks = buildBatchCopTasks(remote_table_ranges);
+            // only send to tiflash node with label [{"engine":"tiflash"}, {"engine-role":"write"}]
+            // pingcap::kv::LabelFilter label_filter = pingcap::kv::labelFilterOnlyTiFlashWriteNode;
+            // FIXME: it depends on another PR, now send request to tiflash node with {"engine"="tiflash"}
+            auto label_filter = pingcap::kv::labelFilterNoTiFlashWriteNode;
+            auto batch_cop_tasks = buildBatchCopTasks(remote_table_ranges, label_filter);
```

Comment:
depends on https://github.com/pingcap/tiflash/pull/6986

Reply:
Is this caused by https://github.com/pingcap/tiflash/pull/6970?

Reply:
yes, because we need to support non-S3-base TiFlash replica and S3-base TiFlash replica in 1 TiDB cluster